doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -185,14 +168,117 @@ Traceability to Reference Model.
 | `req.sec.zon.04` | Zoning | The Architecture **must** support integration with Corporate Identity Management systems. | |
 | `req.sec.cmp.02` | Compliance | The Architecture **must** comply with all applicable standards and regulations. | |
 | `req.sec.cmp.03` | Compliance | The Architecture **must** comply with all applicable regional standards and regulations. | |
-<!-- | `req.sec.ntw.01` | Networking | The Architecture **must** have the underlay network include strong access controls that comply with the applicable security standard (national, regional), for example ISO27001. | |
-| `req.sec.ntw.02` | Networking | The Architecture **must** have all security logs stored in accordance with applicable security standard (national, regional), for example ISO27001. | | -->
 | `req.sec.ntw.03` | Networking | The Architecture **must** have the underlay network incorporate encrypted and/or private communications channels to ensure its security. | |
-|----|--------------|-----------------------------|-------|
 | `req.sec.ntw.04` | Networking | The Architecture **must** configure all of the underlay network components to ensure the complete separation from the overlay customer deployments. | |
 | `req.sec.ntw.05` | Networking | The Architecture **must** have the underlay network include strong access controls that adhere to the V1.1 NIST Cybersecurity Framework. | |
+<!-- | `req.sec.ntw.01` | Networking | The Architecture **must** have the underlay network include strong access controls that comply with the applicable security standard (national, regional), for example ISO27001. | |
+| `req.sec.ntw.02` | Networking | The Architecture **must** have all security logs stored in accordance with applicable security standard (national, regional), for example ISO27001. | | -->
+
 <p align="center"><b>Table 2-8:</b> OpenStack Security Requirements.</p>
 
+
+
+<a name="2.4"></a>
+### 2.4 Architecture and OpenStack Recommendations
+
+The requirements listed in this section are optional, viz. do not need to be implemented in conformant implementations.
+
+2.4.1 General Recommendations
+
+| Ref # | sub-category | Description |  Traceability |
+|----|-------|---------------------|-------------|
+| `req.gen.cnt.01` | Cloud nativeness | The Architecture **should** consist of stateless service components. However, where state is required it must be kept external to the component. | OpenStack consists of both stateless and stateful services where the stateful services utilize a database. For latter see "[Configuring the stateful services](https://docs.openstack.org/ha-guide/control-plane-stateful.html)"|
```

Comment:
__Query__ - @pgoyal01 Should we use `rec.gen.cnt.01` instead of `req.gen.cnt.01` ? A person might end up incrementing the code in requirement section which is already being utilised here or do you want to control at process level?

Reply:
@deebhatia Thanks -- Done. Didn't re-sequence the numbering.